SPECS/dracut/dracut.spec

```diff
@@ -1,48 +1,62 @@
-%define dracutlibdir %{_libdir}/dracut
-%define _unitdir %{_libdir}/systemd/system
+%define dracutlibdir        %{_libdir}/%{name}
+%global __requires_exclude  pkg-config
 
 Summary:        dracut to create initramfs
 Name:           dracut
-Version:        055
-Release:        7%{?dist}
+Version:        059
+Release:        12%{?dist}
 # The entire source code is GPLv2+
 # except install/* which is LGPLv2+
 License:        GPLv2+ AND LGPLv2+
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Group:          System Environment/Base
-URL:            https://dracut.wiki.kernel.org/
-Source0:        http://www.kernel.org/pub/linux/utils/boot/dracut/%{name}-%{version}.tar.xz
-Source1:        https://www.gnu.org/licenses/lgpl-2.1.txt
-Source2:        mkinitrd
-Source3:        megaraid.conf
-Source4:        20overlayfs/module-setup.sh
-Source5:        20overlayfs/overlayfs-mount.sh
-Patch0:         disable-xattr.patch
-Patch1:         fix-initrd-naming-for-mariner.patch
-Patch2:         fix-functions-Avoid-calling-grep-with-PCRE-P.patch
+URL:            https://github.com/dracutdevs/dracut/wiki
+
+Source0:        https://github.com/dracutdevs/dracut/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Source1:        mkinitrd
```

Comment:
I think we should either drop this, or `001-Add-mkinitrd-support-to-dracut.patch`. Best I can tell they are trying to do the same thing.

Reply:
right they are - i'd like to leave it for now because i'd really like someone (i.e. you ;-) to look at both versions to decide which to use.